<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CppEditor::Internal::CppPreProcessorAdditionWidget</class>
 <widget class="QWidget" name="CppEditor::Internal::CppPreProcessorAdditionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>395</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>PP</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="projectLabel">
       <property name="text">
        <string>Project:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="projectComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="separator">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="additionalLabel">
     <property name="text">
      <string>Additional</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TextEditor::SnippetEditorWidget" name="additionalEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEditor::SnippetEditorWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>texteditor/snippets/snippeteditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
